keyboards/jlw/vault35_wkl_universal/info.json

```diff
@@ -0,0 +1,769 @@
+            {"matrix": [0, 10], "x": 195, "y": 13, "flags": 4},
+            {"matrix": [1, 10], "x": 195, "y": 23, "flags": 4},
+            {"matrix": [1, 10], "x": 195, "y": 32, "flags": 4},
+            {"matrix": [1, 10], "x": 195, "y": 43, "flags": 4},
+            {"matrix": [2, 10], "x": 195, "y": 55, "flags": 4},
+            {"matrix": [2, 10], "x": 195, "y": 67, "flags": 4},
+            {"matrix": [3, 10], "x": 190, "y": 79, "flags": 4},
+            {"matrix": [3, 10], "x": 195, "y": 87, "flags": 4},
+            {"matrix": [3, 10], "x": 185, "y": 88, "flags": 4},
+            {"matrix": [3, 9], "x": 176, "y": 88, "flags": 4},
+            {"matrix": [3, 9], "x": 171, "y": 82, "flags": 4},
+            {"matrix": [2, 9], "x": 171, "y": 73, "flags": 4},
+            {"matrix": [2, 9], "x": 170, "y": 67, "flags": 4},
+            {"matrix": [2, 9], "x": 164, "y": 67, "flags": 4},
+            {"matrix": [2, 8], "x": 157, "y": 67, "flags": 4},
+            {"matrix": [2, 8], "x": 151, "y": 67, "flags": 4},
+            {"matrix": [3, 8], "x": 150, "y": 74, "flags": 4},
+            {"matrix": [2, 8], "x": 150, "y": 81, "flags": 4},
+            {"matrix": [3, 8], "x": 147, "y": 88, "flags": 4},
+            {"matrix": [3, 7], "x": 139, "y": 88, "flags": 4},
+            {"matrix": [2, 7], "x": 130, "y": 88, "flags": 4},
+            {"matrix": [3, 7], "x": 122, "y": 88, "flags": 4},
+            {"matrix": [3, 6], "x": 113, "y": 88, "flags": 4},
+            {"matrix": [2, 5], "x": 102, "y": 85, "flags": 4},
+            {"matrix": [3, 5], "x": 94, "y": 85, "flags": 4},
+            {"matrix": [3, 4], "x": 85, "y": 88, "flags": 4},
+            {"matrix": [2, 3], "x": 75, "y": 88, "flags": 4},
+            {"matrix": [3, 3], "x": 66, "y": 88, "flags": 4},
+            {"matrix": [2, 2], "x": 57, "y": 88, "flags": 4},
+            {"matrix": [3, 2], "x": 49, "y": 88, "flags": 4},
+            {"matrix": [3, 2], "x": 46, "y": 83, "flags": 4},
+            {"matrix": [3, 2], "x": 46, "y": 75, "flags": 4},
+            {"matrix": [2, 1], "x": 45, "y": 67, "flags": 4},
+            {"matrix": [2, 1], "x": 39, "y": 67, "flags": 4},
+            {"matrix": [2, 1], "x": 33, "y": 67, "flags": 4},
+            {"matrix": [2, 1], "x": 26, "y": 67, "flags": 4},
+            {"matrix": [3, 0], "x": 25, "y": 73, "flags": 4},
+            {"matrix": [3, 0], "x": 25, "y": 82, "flags": 4},
+            {"matrix": [3, 0], "x": 21, "y": 88, "flags": 4},
+            {"matrix": [3, 0], "x": 12, "y": 88, "flags": 4},
+            {"matrix": [3, 0], "x": 1, "y": 88, "flags": 4},
+            {"matrix": [3, 0], "x": 6, "y": 80, "flags": 4},
+            {"matrix": [3, 0], "x": 1, "y": 70, "flags": 4},
+            {"matrix": [2, 0], "x": 1, "y": 58, "flags": 4},
+            {"matrix": [2, 0], "x": 1, "y": 46, "flags": 4},
+            {"matrix": [1, 0], "x": 1, "y": 34, "flags": 4}
+        ],
+        "max_brightness": 255,
```

Comment:
```suggestion
```

Reply:
Please do not mark things as resolved without addressing them. This is to be removed, as it is the default value and does not need to be redefined.